### PR TITLE
Made the logviewer more accessible 

### PR DESCRIPTION
**Pull request by RachBreeze**
1. By adding the name of the log viewer to the title of the page
2. Changing most links with a click action to buttons 
3. Adding aria-described by to help buttons be more informative to the screen reader user, where they have been styled visually.

Note that the "Save Search" isn't localised in the aria describe by, this replicates the popup which is also not localised.

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/logviewer/search.html [hunk 1]
```diff
@@ -104,115 +104,117 @@
...
+                                                        <button class="btn-reset" ng-switch-when="MachineName" ng-click="vm.findItem(key, val.Value)" localize="title" title="@logViewer_findLogsWithMachineName" aria-describedby="{{key}}">{{val.Value}} <i class="icon-search" aria-hidden="true"></i><p class="sr-only" id="{{key}}"><localize key="logViewer_findLogsWithMachineName"></localize>&nbsp;{{val.Value}}</p></button>
+                                                        <button class="btn-reset" ng-switch-when="RequestUrl" href="{{val.Value}}" target="_blank" rel="noopener" localize="title" title="@logViewer_Open" aria-describedby="{{key}}">{{val.Value}} <i class="icon-link" aria-hidden="true"></i><p class="sr-only" id="{{key}}"><localize key="logViewer_Open"></localize>&nbsp;{{val.Value}}</p></button>
```
[the remark this answers is not in the document]
Comment (RachBreeze) [reply src/Umbraco.Web.UI.Client/src/views/logviewer/search.html]: That makes sense I think by the time I got to this line I was in button all the things mode! ;-)

**Comment by RachBreeze** [comment]
> Ah, don't mind me, I see what's happening. Is this something that will need to be added to new PRs as well to be able to update the title for anything else you might want to improve? I just want to make sure we don't have to repeat this code over and over again everywhere 😄

There is still some work to do around titles in the back office, but it's getting fewer and fewer.  I'm with you on not repeating code over and over

**Comment by RachBreeze** [comment]
Hi @nul800sebastiaan 
I've updated the styling of the buttons on the properties table and fixed a spacing issue around the entry:
`<div ng-if="log.Exception" class="exception">`

The source context now wraps:
image

I've taken a look at the spacing issue and it appears on Umbraco Cloud 8.6.4 but it doesn't look to be this commit.  The screenshot below shows the 8.6.4 site at the top and my localhost at the bottom.

image

Also the styling of the exception has remained unchanged:

image

**Comment by RachBreeze** [comment]
Further investigations needed following feedback from @poornimanayar , so hold this thought thank you

**Comment by nul800sebastiaan** [comment]
Thanks again @RachBreeze - this now looked mostly good! 
Make sure to check my 2 additional commits: 
https://github.com/umbraco/Umbraco-CMS/pull/8213/commits/fd32648f29644d39baa48c4155479b95ccf21e34
https://github.com/umbraco/Umbraco-CMS/pull/8213/commits/301f73f81d8d990f3ecc3396bda64c3e5a3764cd

Apparently an `<a>` always needs a `href` attribute. Make sure to not remove the `rel` attribute either unless you have a specific reason for that.

Additionally, always add `type="button"` to `<button>` elements otherwise they display the behavior of submitting any `<form>` on the page and lastly all `<i>`  elements should get `aria-hidden="true"` since screen readers don't read these properly.

I've made these changes for you now so don't worry about that for this PR, all good, merging! 👍 

Thanks again, top job!! 🏅

**Pull request merged**